src/cover/NoTabsCover.qml

```diff
@@ -13,13 +13,6 @@ import QtQuick 2.1
 import Sailfish.Silica 1.0
 
 CoverBackground {
-    CoverActionList {
-        CoverAction {
-            iconSource: "image://theme/icon-cover-new"
-            onTriggered: window.newTab()
```

Comment:
signal in browser.qml gets deprecated.

Thus, you can also remove the Connections done browser.qml.

FavoriteGrid similar looking signal but that is connected to the tabModel.newTab directly.